cmd/clusterctl/validation/validate_cluster_api_objects_test.go

```diff
@@ -76,34 +74,38 @@ func getNodeWithReadyStatus(nodeName string, nodeReadyStatus v1.ConditionStatus)
 }
 
 func TestGetClusterObjectWithNoCluster(t *testing.T) {
-	g := gomega.NewGomegaWithT(t)
-
 	// Setup the Manager and Controller.
 	mgr, err := manager.New(cfg, manager.Options{})
-	g.Expect(err).NotTo(gomega.HaveOccurred())
+	if err != nil {
+		t.Fatalf("error creating new manager: %v", err)
+	}
 	c = mgr.GetClient()
-	defer close(StartTestManager(mgr, g))
+	defer close(StartTestManager(mgr, t))
 
 	_, err = getClusterObject(c, "test-cluster", "get-cluster-object-with-no-cluster")
-	g.Expect(err).To(gomega.HaveOccurred())
+	if err == nil {
```

Comment:
Checking err can be "cuddled" with the call in places where the call doesn't return anything that needs to be used outside of the conditional check:

`if _, err := getClusterObject(...); err == nil { ... }`

Reply:
I like cuddling too! I often go to a new line when the present one is already pretty long, but I can easily make this change.